README.md

```diff
@@ -3,7 +3,70 @@
 ## Dependencies
 You must install these dependencies on your own: sqlite3, jsoncpp, qt5
 
-Ubuntu example:
-```sh
+### All-in-one
+#### apt-get
+```bash
 sudo apt-get install libsqlite3-dev libjsoncpp-dev qt5-default
```

Comment:
to all-in-one bym jakoś bardziej wyeksponował, żeby było lepiej widac, ze nie trzeba robic calej tej instrukcji